### PR TITLE
Replaces all instances of `Dict` with `Keyword`

### PR DESCRIPTION
**Pull request by victorsolis**

### CODE REVIEW

File: lib/scrivener/html.ex [hunk 1]
```diff
@@ -189,7 +189,7 @@ defmodule Scrivener.HTML do
 
   # Foundation for Sites 6.x implementation
   defp _pagination_links(paginator, [view_style: :foundation, path: path, args: args, params: params]) do
-    url_params = Dict.drop params, Dict.keys(@raw_defaults)
+    url_params = Keyword.drop params, Dict.keys(@raw_defaults)
```
Comment (mgwidmann) [inline lib/scrivener/html.ex]: `Keyword.keys(@raw_defaults)`?

**Comment by mgwidmann** [comment]
Has this been removed completely yet in any version of Elixir yet?

File: lib/scrivener/html.ex [hunk 2]
```diff
@@ -99,11 +99,11 @@ defmodule Scrivener.HTML do
...
   """
   def pagination_links(conn, paginator, args, opts) do
-    opts = Dict.merge opts, view_style: opts[:view_style] || Application.get_env(:scrivener_html, :view_style, :bootstrap)
-    merged_opts = Dict.merge @defaults, opts
+    opts = Keyword.merge opts, view_style: opts[:view_style] || Application.get_env(:scrivener_html, :view_style, :bootstrap)
+    merged_opts = Keyword.merge @defaults, opts
 
     path = opts[:path] || find_path_fn(conn && paginator.entries, args)
-    params = Dict.drop opts, (Dict.keys(@defaults) ++ [:path])
+    params = Keyword.drop opts, (Dict.keys(@defaults) ++ [:path])
```
Comment (mgwidmann) [inline lib/scrivener/html.ex]: `Keyword.keys(@defaults)`?

File: lib/scrivener/html.ex [hunk 3]
```diff
@@ -142,7 +142,7 @@ defmodule Scrivener.HTML do
 
   # Bootstrap implementation
   defp _pagination_links(paginator, [view_style: :bootstrap, path: path, args: args, params: params]) do
-    url_params = Dict.drop params, Dict.keys(@raw_defaults)
+    url_params = Keyword.drop params, Dict.keys(@raw_defaults)
```
Comment (mgwidmann) [inline lib/scrivener/html.ex]: `Keyword.keys(@raw_defaults)` here as well

File: lib/scrivener/html.ex [hunk 4]
```diff
@@ -167,7 +167,7 @@ defmodule Scrivener.HTML do
 
   # Semantic UI implementation
   defp _pagination_links(paginator, [view_style: :semantic, path: path, args: args, params: params]) do
-    url_params = Dict.drop params, Dict.keys(@raw_defaults)
+    url_params = Keyword.drop params, Dict.keys(@raw_defaults)
```
Comment (mgwidmann) [inline lib/scrivener/html.ex]: :wink:

**Comment by mgwidmann** [comment]
Thanks a bunch!

**Comment by victorsolis** [comment]
Wow. Must've missed `g` lol. Sorry about that. Give me a sec.

And nope, just deprecated. Probbably in 2.0.

**Comment by mgwidmann** [comment]
Awesome thanks!

**Pull request merged**